event-log/src/main/scala/io/renku/eventlog/statuschange/commands/ToTriplesGenerated.scala

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2020 Swiss Data Science Center (SDSC)
+ * A partnership between École Polytechnique Fédérale de Lausanne (EPFL) and
+ * Eidgenössische Technische Hochschule Zürich (ETHZ).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.renku.eventlog.statuschange.commands
+
+import cats.effect.Bracket
+import cats.syntax.all._
+import ch.datascience.db.{DbTransactor, SqlQuery}
+import ch.datascience.graph.model.events.EventStatus.{GeneratingTriples, TriplesGenerated}
+import ch.datascience.graph.model.events.{CompoundEventId, EventStatus}
+import ch.datascience.graph.model.{events, projects}
+import ch.datascience.metrics.LabeledGauge
+import doobie.implicits._
+import eu.timepit.refined.auto._
+import io.renku.eventlog.EventLogDB
+import io.renku.eventlog.statuschange.commands.ProjectPathFinder.findProjectPath
+
+import java.time.Instant
+
+final case class ToTriplesGenerated[Interpretation[_]](
```

Comment:
Should this be public?

Reply:
So this is not used yet so yes it could be private